shared/provision/select-other-device/index.tsx

```diff
@@ -65,15 +65,17 @@ class SelectOtherDevice extends React.Component<Props> {
         gapEnd={true}
       >
         <Box2 direction="vertical" fullWidth={true} style={styles.contentBox} gap={'medium'}>
-          <Text center={true} type={isMobile ? 'BodyBig' : 'Header'}>
-            For security reasons, you need to authorize with an existing device. Which of your existing
```

Comment:
cc @cecileboucheron should this line to appear on desktop too? it might be confusing to be presented with a list of devices with no explanation otherwise

Reply:
Totally. The desktop design was updated recently to this: https://zpl.io/ank1goQ